### PR TITLE
Update ID 40033 for MongoDB (NoSQL Injection)

### PR DESCRIPTION
**Pull request by LuigiCasciaro**

### CODE REVIEW

File: src/org/zaproxy/zap/model/Tech.java [hunk 1]
```diff
@@ -160,4 +160,4 @@ public int compareTo(Tech o) {
         
         return this.toString().compareTo(o.toString());
     }
-}
+}
```
Comment (kingthorin) [inline src/org/zaproxy/zap/model/Tech.java]: Could you revert this or exclude this file?

**Comment by thc202** [comment]
@LuigiCasciaro note that the commits are using different accounts (or not tied to your own).

File: src/org/zaproxy/zap/model/Tech.java (hunk 1 above)
Comment (LuigiCasciaro) [reply src/org/zaproxy/zap/model/Tech.java]: Of course, but it's strange because it does not change anything between the red and green lines.

Comment (thc202) [reply src/org/zaproxy/zap/model/Tech.java]: It removed a newline.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**